tests/core/test_daemon_rpc.py

```diff
@@ -3,15 +3,14 @@
 
 from tests.setup_nodes import setup_daemon
 from chia.daemon.client import connect_to_daemon
-from tests.setup_nodes import bt
 from chia import __version__
 from tests.util.socket import find_available_listen_port
 from chia.util.config import save_config
 
 
 class TestDaemonRpc:
     @pytest_asyncio.fixture(scope="function")
-    async def get_daemon(self):
+    async def get_daemon(self, bt):
         bt._config["daemon_port"] = find_available_listen_port()
```

Comment:
We may not need it in the sense of "is this necessary". But fixing this would be a further improvement, and out of the (massive) scope of this patch, because `setup_daemon(btools)` in `tests/setup_nodes.py` already too a BlockTools as an argument. So, we either have to understand and change `setup_daemon`, or we have toadd a parameter to `get_daemon` to compensate for the global that has been removed.

Reply:
Take a look at the change history of `setup_daemon` to see what I mean.

Oh, wait. When you say "we don't need to do this anymore", do you mean:
`    bt._config["daemon_port"] = find_available_listen_port()` ?

I thought you meant adding `bt` as an argument.
Sure, I'll take a look at that config assignment - I don't think we need it.

Reply:
Okay, take a look at that now.